web-test-runner.config.js

```diff
@@ -0,0 +1,17 @@
+
+module.exports = config => {
+    return {  
```

Comment:
Delete unnecessary space.
```suggestion
    return {
```